config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [] do
+    resources :posts, only: %i[index show], module: :users
+  end
```

Comment:
- [ ] [OPTIONAL] Good job using [resource routing](https://guides.rubyonrails.org/v3.2.2/routing.html#resource-routing-the-rails-default) for defining routes. It not only simplifies your code but also aligns with Rails conventions, making your application more maintainable and developer-friendly. like this:


```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```